build/testdata/052.golden

```diff
@@ -1,3 +1,11 @@
+# Unary operators
+
+-foo
+
++foo
```

Comment:
add test for `+ + foo` or `- + foo`

Reply:
Done, also added support for bitwise operators (in the same PR)